server/ngwmn/templates/site_location.html

```diff
@@ -30,7 +30,7 @@ <h1>{{ feature.SITE_NAME }}</h1>
             </tr>
             <tr>
                 <td scope="row">Lat/Long ({{ feature.HORZ_DATUM }})</td>
-                <td>{{ _(feature.DEC_LAT_VA) }}, {{ feature.DEC_LONG_VA }}</td>
+                <td>{{ _(feature.DEC_LAT_VA | round(4)) }}, {{ _(feature.DEC_LONG_VA | round(4)) }}</td>
```

Comment:
I'm wondering if it's worthwhile to add tests to verify that all the `feature.BLAH` things show up. Not necessary though.